### PR TITLE
1.1.3

### PR DESCRIPTION
**Pull request by brianmcmichael**
1.1.3

Kovan and Mainnet

* Adds address for `MCD_JOIN_ETH_B`
* Adds address for `MCD_FLIP_ETH_B`

### CODE REVIEW

File: releases/kovan/active [hunk 1]
```diff
@@ -1 +1 @@
-1.1.2
\ No newline at end of file
+1.1.3
```
Comment (zdumitru) [inline releases/kovan/active:1]: Didn't we agree that the active link is updated only after the spell is cast?

**Review by zdumitru — CHANGES_REQUESTED** [review]

File: releases/kovan/active (hunk 1 above)
Comment (zdumitru) [reply releases/kovan/active:1]: Ah, that's kovan. All good then.

Comment (EdNoepel) [reply releases/kovan/active:1]: For Kovan though?

Comment (brianmcmichael) [reply releases/kovan/active:1]: Ah, I just updated the symlink for completeness. I am pretty sure I updated mainnet also. i can undo it, but we'll need to remember to update this once it's cast.

**Review by zdumitru — CHANGES_REQUESTED** [review]
Please revert mainnet back to 1.1.2 and I will remember to update it to 1.1.3 when (if) the voting approves it and the spell is cast.

**Comment by brianmcmichael** [comment]
* reverted mainnet `active` to 1.1.2 for now.
* added `CHANGELOG` address to kovan and mainnet 1.1.3
* Updated IPFS links for new json
* Added ChainLog.abi to kovan and mainnet zip files

File: releases/mainnet/active [hunk 2]
```diff
@@ -1 +1 @@
-1.1.2
\ No newline at end of file
+1.1.3
```
[the remark this answers is not in the document]
Comment (brianmcmichael) [reply releases/mainnet/active]: Updated.

**Review by zdumitru — APPROVED** [review]

**Review by kmbarry1 — APPROVED** [review]
LGTM

At some point we should standardize on a canonical ilk ordering (probably alphabetical). Hopefully no one depends on the order they appear in the changelog...

**Pull request merged**